index.js

```diff
@@ -156,6 +156,10 @@ const runChecks = async (context, checkEvent) => {
           case constants.oldBuildLabelCheck:
             callable.push(staleBuildModule.removeOldBuildLabel(context));
             break;
+          case constants.ensureNewIssuesHaveProjectsCheck:
+            callable.push(periodicCheckModule.
+              ensureNewIssuesHaveProjects(context));

```

Comment:
```suggestion
            callable.push(
              periodicCheckModule.ensureNewIssuesHaveProjects(context));

```

Reply:
Will change them in the latest commit Sir.Thanks.